testcases/android/master.jinja2

```diff
@@ -0,0 +1,51 @@
+{% extends "master.jinja2" %}
+
+{# timeouts #}
+{% set timeout_download = timeout_download|default(10) %}
+{% set timeout_between_download_deploy = timeout_between_download_deploy|default(0) %}
+{% set timeout_deploy = timeout_deploy|default(25) %}
+{% set timeout_between_deploy_boot = timeout_between_deploy_boot|default(0) %}
+{% set timeout_boot = TARGET_BOOT_TIMEOUT|default(10) %}
```

Comment:
Whats the rational behind inventing your own variables?

Reply:
will update to use TARGET_BOOT_TIMEOUT